examples/example_composed_target_function.py

```diff
@@ -0,0 +1,160 @@
+"""
```

Comment:
The example is lacking the CLI of the other examples (you can just copy the code) and explanatory docstrings that can be picked up by sphinx.